oarepo_requests/utils.py

```diff
@@ -22,3 +28,92 @@ def get_allowed_request_types(queryied_record_cls):
                 ret[request_name] = request_type
                 break
     return ret
+
+
+def request_exists(
+    identity,
+    topic,
+    type_id,
+    topic_type="thesis_draft",
+    receiver_type=None,
+    receiver_id=None,
+    creator_type=None,
+    creator_id=None,
+    topic_id=None,
+    add_args=None,
+):
+    """Return the request id if an open request already exists, else None."""
+
+    must = [
+        dsl.Q("term", **{"type": type_id}),
+        dsl.Q("term", **{"is_open": True}),
+    ]
```

Comment:
This is always open, do we need the two methods?
Also, topic_type should not be "thesis_draft" as this is only an example